src/main/java/net/minecraftforge/client/model/generators/BlockStateProvider.java

```diff
@@ -423,7 +445,7 @@ public void wallBlock(WallBlock block, String name, ResourceLocation texture) {
     private void wallBlockInternal(WallBlock block, String baseName, ResourceLocation texture) {
         wallBlock(block, models().wallPost(baseName + "_post", texture), models().wallSide(baseName + "_side", texture), models().wallSideTall(baseName + "_side_tall", texture));
     }
-    
+
```

Comment:
Could you remove this whitespace change?

Reply:
Done